deploy/azure/app/kube/main.tf

```diff
@@ -15,7 +15,7 @@ module "default_label" {
 }
 
 module "app" {
-  source                               = "git::https://github.com/amido/stacks-terraform//azurerm/modules/azurerm-server-side-app?ref=v1.5.1"
+  source                               = "git::https://github.com/ensono/stacks-terraform//azurerm/modules/azurerm-server-side-app?ref=v1.5.1"
```

Comment:
Should this have a double / in the middle? 
`stacks-terraform//azurerm`

Reply:
Yeah that is normal for Terraform modules. Everything before the `//` is the repo and everything after is the path to the file.

https://developer.hashicorp.com/terraform/language/modules/sources#modules-in-package-sub-directories